src/app/linux/.gitignore

```diff
@@ -0,0 +1 @@
+flutter/ephemeral
```

Comment:
Do we want to include linux & macos in the template?

Reply:
and also web

Reply:
I think so. It's not heavy and doesn't cause any issue like an Uno project.